.github/workflows/test_ci.yml

```diff
@@ -60,26 +85,16 @@ jobs:
         with:
           node-version: "14"
 
-      - name: Cache Node Modules
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
+      - name: Download the build folders
+        uses: actions/download-artifact@v3
         with:
-          # npm cache files are stored in `~/.npm` on Linux/macOS
-          path: ~/.npm
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-
-      - name: Install Packages
-        run: npm ci
-        env:
-          CYPRESS_CACHE_FOLDER: ${{ github.workspace }}/node_modules/.cache/Cypress
+          name: build
+          path: build
 
       - name: Integration - Synpress
         if: ${{ !contains(github.event.head_commit.message, '[skip test]') }}
-        run: npm run integration:headful
+        run: npm run integration:headful:ci
         env:
+          CYPRESS_PROJECT_ID: "trkg83"
+          CYPRESS_RECORD_KEY: "5fd3babb-9ca9-4743-a9c5-1e8164dbf9ef"
```

Comment:
ooo, did u use dlt email for this?

Reply:
nono will need to replace this with the one form dlt email